hiss/volunteer/views.py

```diff
@@ -63,6 +63,27 @@ def post(self, request: Request, format: str = None):
         return response.Response(status=status.HTTP_200_OK)
 
 
+class ListDietaryRestrictionsView(views.APIView):
+    """
+    Lists all of the available DietaryRestrictions
+    """
+
+    permission_classes = [
+        permissions.IsAuthenticated & (IsVolunteer | permissions.IsAdminUser)
```

Comment:
Why do we not need ```permissions.``` preceding ```IsVolunteer``` like we have for ```permissions.IsAuthenticaded``` and ```permissions.IsAdminUser```?

Reply:
`IsVolunteer` is a custom permission that was built specifically for the registration system. See the imports at the top for more information